modules/deploy/partials/requirements.adoc

```diff
@@ -9,6 +9,8 @@ endif::[]
 
 - Minimum version required of Ubuntu: {supported-ubuntu-required}. *Recommended*: {supported-ubuntu-recommended}
 
+NOTE: Enabling SELinux (Security-Enhanced Linux) can result in latency issues. If you wish to avoid such latency issues, do not use this mechanism.
```

Comment:
```suggestion
NOTE: Enabling SELinux (Security-Enhanced Linux) can result in latency issues. If you wish to avoid such latency issues, do not enable it in your system.
```

Looks like SELinux might be part of the kernel. So it's a matter of enabling/disabling it.